python/lsst/obs/decam/decamMapper.py

```diff
@@ -219,11 +220,6 @@ def bypass_instcal(self, datasetType, pythonType, butlerLocation, dataId):
         for kw in ('LTV1', 'LTV2'):
             md.remove(kw)
 
-        # Remove TPV keywords from the metadata
-        for kw in md.paramNames():
-            if re.match(r'PV\d_\d', kw):
-                md.remove(kw)
-
```

Comment:
That is an excellent point. I removed it because the new SkyWcs class handles TPV natively, but I'm not actually sure if that's why these were removed. I have made that a separate commit and will ask about it (unless you know?).

Reply:
I believe the `TPV` keywords broke afw. @RobertLuptonTheGood or @hsinfang will know.

Reply:
Thanks Paul...I asked for a comment because I haven’t got a clue!

Reply:
We stripped the TPV keywords because wcslib would (correctly) interpret them, but we could not write TPV or easily convert to SIP, and the feeling was that breaking round-tripping in that way was undesirable (DM-3793). There was also an issue where TPV headers would leak from instcals into calexps via HDU 0, and the most expedient way to stop that was by deleting all TPV headers (DM-8042). There is a nominal test for this in `testGetInstcal.py`, but it might be worth checking a calexp for any inconsistencies.